zedd-app/src/components/AppBody.tsx

```diff
@@ -49,6 +55,88 @@ const useStyles = makeStyles((theme) => ({
   },
 }))
 
+export function DateRangePicker(state: AppState) {
+  const [anchorEl, setAnchorEl] = React.useState(null)
+
+  const handleClick = (event: any) => {
+    setAnchorEl(event.currentTarget)
+  }
+
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+
+  const open = Boolean(anchorEl)
+  const id = open ? 'simple-popover' : undefined
+
+  const Picker = () => (
+    <DateRange
+      months={2}
+      editableDateInputs={true}
+      onChange={(item) => {
+        state.showing = { start: item.selection.startDate!, end: item.selection.endDate! }
+      }}
+      moveRangeOnFirstSelection={true}
+      ranges={[
+        {
+          startDate: state.showing.start as Date,
+          endDate: state.showing.end as Date,
+          key: 'selection',
+        },
+      ]}
+    />
+  )
+  const Closed = () => (
+    <div
+      className='MuiFormControl-root MuiTextField-root'
+      style={{ width: '100%', minWidth: '20rem' }}
+    >
+      <label
+        className='MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled'
+        data-shrink='true'
+      >
+        Start 🡢 End
+      </label>
+      <div className='MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl'>
+        <input
+          aria-invalid='false'
+          readOnly
+          type='text'
+          className='MuiInputBase-input MuiInput-input'
+          value={
+            formatDate(state.showing.start as Date, 'E, do MMMM') +
+            ' 🡢 ' +
+            formatDate(state.showing.end as Date, 'E, do MMMM')
+          }
+        />
+      </div>
+    </div>
+  )
+  return (
+    <div>
+      <div aria-describedby={id} onClick={handleClick}>
+        <Closed />
```

Comment:
As the name "Closed" is not particularly descriptive and only used in one place, we can inline it.